src/deepsparse/evaluation/registry.py

```diff
@@ -11,12 +11,71 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
+"""
+Implementation of a registry for evaluation functions
+"""
+from typing import Any, Callable, List, Optional, Union
 
 from sparsezoo.utils.registry import RegistryMixin
 
 
 __all__ = ["EvaluationRegistry"]
 
 
+def _merge_name_alias(
+    name: Optional[str], alias: Union[str, List[str], None]
+) -> List[str]:
+    """
+    Merge the name and alias into a single list of names
+    """
+    name = [name]
+    if alias:
+        if not isinstance(alias, list):
+            alias = [alias]
+        name.extend(alias)
+    return name
+
+
 class EvaluationRegistry(RegistryMixin):
-    pass
+    """
+    Extends the RegistryMixin to enable registering and loading of evaluation
+    functions. Adds the ability to register a function under multiple names.
+    Since functions are stateless, we can register the same function under
+    multiple names and load it from the registry using any of the names.
```

Comment:
From what I can tell we wouldn't need this child class if `RegistryMixin` had the capability for multiple aliases? It might be cleaner to implement all the registry code in SparseZoo